data/human/culture conversations.txt

```diff
@@ -791,3 +791,13 @@ mission "Raider returning to Mordente-Bridi"
 			`Watching the microgravity section of the spaceport from one of the bars in Mordente Cylinder, you observe a Bastion in pirate markings coming nearly too fast for safe approach to the station, its hull covered in gashes and burns from what looks to be Plasma Cannon shots.`
 			`	As it comes to a stop in a maintenance bay, the spaceport's engineering shuttles appear from all over and begin buzzing around the Bastion, assessing the damage and beginning the work on repairs. Meanwhile, its crew disperse from the ship and immediately begin unloading a multitude of busted-up crates and containers from the hold, probably raided from a civilian transport. It looks like their haul was worth the damage.`
 			`	Within the hour the work shuttles scatter from the ship as they finish patching up its battered hull, and the Bastion maneuvers out of the bay almost as haphazardly as it did when it entered.`
+
+mission "Militia returning to Solace"
+	source "Solace"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`Wandering around the main spaceport on Solace, you notice that Delta V is building a private spaceport near their main company offices. It seems that the work has been going on for a long while, but has only just been unveiled, because there is a crowd of interested spectators near the perimeter of the fence.`
+			`	Suddenly an Argosy comes streaking out of the sky, almost crashing into smaller ships in the main spaceport before swerving toward the new rows of landing pads. There are shouts from the throng of onlookers, who seem to think this was a stunt. Your more practiced eye, however, tells you that there are bullet marks on the Argosy's hull from a Gatling Gun, and it was probably a miscalculation by the pilot's computer that almost caused a collision.`
```

Comment:
"rows of landing pads" works but is a bit pedantic. Maybe just:
```suggestion
			`	Suddenly an Argosy comes streaking out of the sky, almost crashing into smaller ships in the main spaceport before swerving toward the new landing pads. There are shouts from the throng of onlookers, who seem to think this was a stunt. Your more practiced eye, however, tells you that there are bullet marks on the Argosy's hull from a Gatling Gun, and it was probably a miscalculation by the pilot's computer that almost caused a collision.`
```

Also, the last sentence is a little bit run-ony. It's probably okay, but you might want to break it up somewhere.